chirp/drivers/ft450d.py

```diff
@@ -1165,206 +1016,240 @@ def _do_panel_settings(self, pnlset):    # - - - Panel settings
+        tmp = "SEL dial 2nd function (push)"
+        rs = RadioSetting("seldial", tmp, rx)
         pnlset.append(rs)
     # End _do_panel_settings
 
     def _do_panel_buttons(self, pnlcfg):      #- - - Current Panel Config
         _settings = self._memobj.settings
 
-        rs = RadioSetting("pnl_cs", "C.S. Function",
-                          RadioSettingValueList(self.FUNC_LIST,
-                          self.FUNC_LIST[_settings.pnl_cs]))
+        optn = self.FUNC_LIST[_settings.pnl_cs]
+        rx = RadioSettingValueList(self.FUNC_LIST, optn)
+        tmp = "C.S. Function"
+        rs = RadioSetting("pnl_cs", tmp, rx)
         pnlcfg.append(rs)
 
-        rs = RadioSetting("nb", "Noise blanker",
-                          RadioSettingValueBoolean(_settings.nb))
+        rx = RadioSettingValueBoolean(_settings.nb)
+        tmp = "Noise blanker"
+        rs = RadioSetting("nb", tmp, rx)
         pnlcfg.append(rs)
 
         options = ["Auto", "Fast",  "Slow", "Auto/Fast", "Auto/Slow", "?5?"]
-        rs = RadioSetting("agc", "AGC",
-                          RadioSettingValueList(options,
-                          options[_settings.agc]))
+        optn = options[_settings.agc]
+        rx = RadioSettingValueList(options, optn)
+        tmp = "AGC"
+        rs = RadioSetting("agc", tmp, rx)
         pnlcfg.append(rs)
 
-        rs = RadioSetting("keyer", "Keyer",
-                          RadioSettingValueBoolean(_settings.keyer))
+        rx = RadioSettingValueBoolean(_settings.keyer)
+        tmp = "Keyer"
+        rs = RadioSetting("keyer", tmp, rx)
         pnlcfg.append(rs)
 
-        rs = RadioSetting("fast", "Fast step",
-                          RadioSettingValueBoolean(_settings.fast))
+        rx = RadioSettingValueBoolean(_settings.fast)
+        tmp = "Fast step"
+        rs = RadioSetting("fast", tmp, rx)
         pnlcfg.append(rs)
 
-        rs = RadioSetting("lock", "Lock (per Lock Mode)",
-                          RadioSettingValueBoolean(_settings.lock))
+        rx = RadioSettingValueBoolean(_settings.lock)
+        tmp = "Lock (per Lock Mode)"
+        rs = RadioSetting("lock", tmp, rx)
         pnlcfg.append(rs)
 
         options = ["PO",  "ALC", "SWR"]
-        rs = RadioSetting("mtr_mode", "S-Meter mode",
-                          RadioSettingValueList(options,
-                          options[_settings.mtr_mode]))
+        optn = options[_settings.mtr_mode]
+        rx = RadioSettingValueList(options, optn)
+        tmp = "S-Meter mode"
+        rs = RadioSetting("mtr_mode", tmp, rx)
         pnlcfg.append(rs)
         # End _do_panel_Buttons
 
     def _do_vox_settings(self, voxdat):     # - - VOX and DATA Settings
         _settings = self._memobj.settings
 
-        rs = RadioSetting("vox_dly", "VOX delay (x 100 ms)",
-                          RadioSettingValueInteger(1, 30, _settings.vox_dly))
+        rx = RadioSettingValueInteger(1, 30, _settings.vox_dly)
+        tmp = "VOX delay (x 100 ms)"
```

Comment:
I find this confusing too: is the "x" a multiplication like the "*" in another string?